packages/tpetra/core/test/KokkosIntegration/KokkosUVM.cpp

```diff
@@ -135,7 +135,7 @@ void reportTimings(Teuchos::RCP<Teuchos::StackedTimer> timer) {
 }
 
 int main(int argc, char *argv[]) {
-  Kokkos::ScopeGuard kokkos(argc, argv);
+  Kokkos::initialize(argc, argv);
```

Comment:
why doesn't Kokkos::ScopeGuard work anymore?

Reply:
It probably does, actually.  I've just never used it before and removed it trying to debug something else.